### DISCUSSION TITLE
AI On the Edge Water Meter

### DISCUSSION THREAD

**Discussion by Phoenix17785** [jomjol/AI-on-the-edge-device#2968] [Q&A]
Hi,

i‘ve setup everything accordingly to documentation.
And I added the program code for it to configuration.yaml.

I’ve also set up Moscito Broker and edited the MQTT Connection on the Web Interface of the Water Meter.

I do see the entities in Home Assistant.
They show an error.

Where can I find the error messages?

![IMG_5477](https://github.com/jomjol/AI-on-the-edge-device/assets/155656072/e802b968-131e-4bae-9c16-40c2a02180b8)
![IMG_5476](https://github.com/jomjol/AI-on-the-edge-device/assets/155656072/35c1f6ed-cad6-4546-bd67-e843d57f3cab)

**Comment by SybexX** [comment]
what do you mean by "Where can I find the error messages?"
Do you mean a more detailed description of the errors?
You get the error “Rate to high” because the difference between the previous value and the read value is too high.
To eliminate the error, you have to set the previous value manually.

**Comment by Phoenix17785** [comment]
Yes. A more detailed error message to know where to look at for troubleshooting. Von meinem iPhone gesendetAm 04.03.2024 um 20:01 schrieb michael ***@***.***>:﻿
what do you mean by "Where can I find the error messages?"
Do you mean a more detailed description of the errors?
You get the error “Rate to high” because the difference between the previous value and the read value is too high.
To eliminate the error, you have to set the previous value manually.

**Comment by SybexX** [comment]
I don't know if there is a documentary about it somewhere, you can check the wiki. https://jomjol.github.io/AI-on-the-edge-device-docs/Error-Codes/
Actually there are only two errors that are output during the calculation:
Rate too high - Read:.... >>> Difference between the Raw Value and the Previous Value too high.
Neg. Rate - Read:.... >>> The Raw Value is lower than the Previous Value.
And in both cases, only setting the Previous Value manually helps.

**Comment by SybexX** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/FAQs/#rate-too-high-read
